backend/graphql/answer/type.graphql

```diff
@@ -0,0 +1,9 @@
+type Answer {
+    id: ID!
+    content: String!
+    is_correct: Boolean!
```

Comment:
I think this can be null since it is no longer needed in the mutation.

Reply:
`is_correct ` cannot be null because it was automatically populated to false so I think I don't need to change this line